harvard-university-for-the-creative-arts.csl

```diff
@@ -231,10 +236,15 @@
             <et-al font-style="italic"/>
           </names>
           <group delimiter=", ">
-            <group font-style="normal" font-variant="normal">
+            <group font-style="normal" font-variant="normal" delimiter=" ">
               <text variable="container-title" font-style="italic" suffix="."/>
               <text variable="collection-title" font-style="italic"/>
-              <text variable="volume" prefix=" Vol " suffix="."/>
+              <choose>
+                <if match="any" variable="volume">
+                  <text term="volume" form="short" text-case="capitalize-first"/>
+                  <text variable="volume" text-case="uppercase" prefix=" " suffix="."/>
+                </if>
+              </choose>
```

Comment:
Thanks I didn't realise group acted like that, I think that I've done what you suggested.

really appreciate your help